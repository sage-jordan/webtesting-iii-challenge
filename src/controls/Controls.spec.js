// Test:
//- provide buttons to toggle the `closed` and `locked` states.
// - buttons' text changes to reflect the state the door will be in if clicked
// - the closed toggle button is disabled if the gate is locked
// - the locked toggle button is disabled if the gate is open