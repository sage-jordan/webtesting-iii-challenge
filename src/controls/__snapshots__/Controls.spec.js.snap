// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Control /> matches snapshot 1`] = `
<div
  className="controls panel"
  data-testid="controls"
>
  <button
    className="toggle-btn"
    data-testid="lockedUnlocked"
    disabled={true}
  >
    Lock Gate
  </button>
  <button
    className="toggle-btn"
    data-testid="openClosed"
  >
    Close Gate
  </button>
</div>
`;
